Editor/UI/ProjectAuditorWindow.cs

```diff
@@ -352,6 +353,18 @@ static void OnLoad()
                     },
                     analyticsEvent = (int)ProjectAuditorAnalytics.UIButton.ShaderCompilerMessages
                 });
+
+                ViewDescriptor.Register(new ViewDescriptor
+                {
+                    category = IssueCategory.Texture,
+                    name = "Texture",
+                    menuOrder = 6,
+                    showInfoPanel = true,
+                    // type = typeof(SummaryView),
```

Comment:
please remove commended out code

Reply:
Solved in [Tweaks to Texture Module] commit.